docs/data/base/components/input/input.md

```diff
@@ -40,7 +40,7 @@ The `multiline` prop transforms the `<input>` field into a `<textarea>` element.
 
 {{"demo": "InputMultiline.js"}}
 
-If you want the `<textarea>` to grow with the content, you can use the [`TextareaAutosize`](/components/textarea-autosize/) component.
+If you want the `<textarea>` to grow with the content, you can use the [`TextareaAutosize`](/base/react-textarea-autosize/) component.
```

Comment:
```suggestion
If you want the `<textarea>` to grow with the content, you can use the MUI Base's [`TextareaAutosize`](/base/react-textarea-autosize/) component.
```

I wonder if we shouldn't be more explicit about recommending using components that (even though are from MUI) live in another package. I'm dropping this here based on previous experience where the dev managers were very strict and reluctant to add any sort of new packages/dependencies on the app. 

Not mentioning up-front that the component is in another package might create an expectation that it's only an import away when you'd have to actually install and deal with a new thing. Not sure if I'm worrying too much here though 😅

Reply:
> Not mentioning up-front that the component is in another package might create an expectation that it's only an import away when you'd have to actually install and deal with a new thing

It is just an import away :)

@mui/material depends on @mui/base, so there is no need to install anything else. Let me know if it's not clear from the docs that you can import the component from either @mui/base or @mui/material.

Reply:
What if I'm using Base? Can I import Material things just as well?

Reply:
You will have to `npm install @mui/material` in such case. But why would you want to do so? We're not mentioning any Material components in Base docs, nor we encourage importing from @mui/material AFAIK.

Reply:
Yeah, makes sense. Right! Thank you!